public/javascript/components/surveyController.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Survey Controller
+ */
+
+import $ from 'jquery';
+import firebase from "firebase/app";
+import "firebase/database";
+
+// Submit button on click
+$('.survey__submit').click(function(e) {
+  let getCheckedButtons = getAllRadioButtonsChecked()
+  
+  e.preventDefault()
```

Comment:
Let's move this to before we assign the `getCheckedButtons` 👍

Reply:
👍🏾